### PR TITLE
Make Hide Avatars work on notifications

### LINKED ISSUE #358: Make Block Avatar also block avatars in notifications
As far as I can tell, Block Avatar only works on the dashboard right now (i might be missing some place to change this?). Would love to see it updated to work with notifications as well!

### PR DESCRIPTION
**Pull request by ThePsionic**
Fixes #358.

### CODE REVIEW

File: Extensions/hideavatars.js [hunk 1]
```diff
@@ -28,11 +28,25 @@ XKit.extensions.hideavatars = new Object({
...
+			XKit.extensions.hideavatars.blognames.forEach(function (blogname) {
+				$(".notification a").filter(function() {
+					return $(this).attr("href").slice(7).split(".")[0] === blogname.title;
+				}).find(".avatar").attr("src", hidden_avatar);
+			});
+
+			XKit.extensions.hideavatars.blognames.forEach(function (blogname) {
+				$(".notification a").filter(function() {
+					return $(this).attr("href").slice(7).split(".")[0] === blogname.title;
+				}).find(".avatar").attr("style", "opacity: 0.5");
```
Comment (Wolvan) [inline Extensions/hideavatars.js]: Why did you not just chain the 2 attr calls?
```
.find(".avatar").attr("src", hidden_avatar).attr("style", "opacity: 0.5");
```

Comment (ThePsionic) [inline Extensions/hideavatars.js]: Because I didn't know that was possible. Will fix now ^_^

**Comment by Wolvan** [comment]
Bump the version and you should be good

**Comment by Wolvan** [comment]
Tested and works for me, LGTM
@homu r+

**Pull request merged**